portfolio/src/main/webapp/index.html

```diff
@@ -28,12 +28,16 @@ <h2>Pictures!</h2>
       <p>I was there for a just a couple of days but I had a lot of fun, this is me visiting Marina del Rey's pier.</p>
       <img src="images/beach.jpg" alt="Sunset at sea" class="standard-size image-on-center"/>
       <p>I took this picture on my last day in California, I really enjoyed seeing the waves and the orange color of the sky.</p>
-      <h2>Random Facts about me </h2> 
+      <h2>Random Facts about me</h2> 
       <p>Click this button to see a random fact about me:</p>
       <button onclick="addRandomFact()">Click Here!</button>
       <div id="fact-container"></div>
+      <br><br/>
```

Comment:
Did you mean `<br></br>`? In any case br tags just add a line break immediately, so we always do `<br />`, which is the same as `<br></br>` to save space.

Reply:
Sorry, I should have been more clear. Change to `<br />`. What you changed it to is valid, but it's pointlessly long.

Reply:
thank you, I'll make the changes